lib/_emerge/actions.py

```diff
@@ -2634,13 +2635,55 @@ def apply_priorities(settings):
 	nice(settings)
 
 def nice(settings):
+	nice_value: str = settings.get("PORTAGE_NICENESS", "0")
+
 	try:
-		os.nice(int(settings.get("PORTAGE_NICENESS", "0")))
+		os.nice(int(nice_value))
 	except (OSError, ValueError) as e:
 		out = portage.output.EOutput()
-		out.eerror("Failed to change nice value to '%s'" % \
-			settings.get("PORTAGE_NICENESS", "0"))
+		out.eerror(f"Failed to change nice value to {nice_value}")
+		out.eerror("%s\n" % str(e))
+		return
+
+	sched_autogroup = Path("/proc/sys/kernel/sched_autogroup_enabled")
+	if not sched_autogroup.is_file():
```

Comment:
Why can't we eliminate this stat call?

Reply:
I need a really really really solid argument for why this stat call is necessary.

Reply:
Please have amadio test a patch without the unecessary stat call, and get him to sign off on it.